### ISSUE TITLE
Multi-output (multioutput) regression/classification

### ISSUE THREAD

**Issue by abedidev** [PriorLabs/TabPFN#367]
### Describe the workflow you want to enable

Is there currently any feature that allows using a pre-trained model, or fine-tuning specific layers of it, for multi-output regression or classification? I’ve looked into TabPFN, tabpfn-extensions, and finetune_tabpfn_v2, but couldn’t find such functionality.

Thanks in advance,
Ali

### Describe your proposed solution

I’m not looking to use scikit-learn’s MultiOutputRegressor, I need to train/fine-tune a model that supports multiple outputs.

### Impact

None

**Comment by iivalchev** [comment]
Hi @abedidev, multi-output regression is not supported (at least for now), hence you can't fine-tune out of the box. I guess you can modify the transformer by adding an output decoder which could output multiple logits. However this would be a substantial change and you would need to train this decoder.

For classification TabPFN does support multi-class if this is what you mean, but it doesn't support multiple multi-class predictions.

**Comment by oscarkey** [comment]
I'm combining this with the existing issue on multi-output regression, but if multiple multi-class predictions are very important to you feel free to reopen!
